Orange/widgets/data/owtranspose.py

```diff
@@ -0,0 +1,107 @@
+from AnyQt.QtCore import Qt
+
+from Orange.data import Table, StringVariable
+from Orange.widgets.settings import (Setting, ContextSetting,
+                                     PerfectDomainContextHandler)
+from Orange.widgets.utils.itemmodels import DomainModel
+from Orange.widgets.widget import OWWidget, Msg
+from Orange.widgets import gui
+
+
+class OWTranspose(OWWidget):
+    name = "Transpose"
+    description = "Transpose data table."
+    icon = "icons/Transpose.svg"
+    priority = 2000
+
+    inputs = [("Data", Table, "set_data")]
+    outputs = [("Data", Table)]
+
+    resizing_enabled = False
+    want_main_area = False
+
+    settingsHandler = PerfectDomainContextHandler(metas_in_res=True)
```

Comment:
Why `PerfectDomainContextHandler`? I prefer `DomainContextHandler` when there is no need to match all attributes, because it matches more domains.